packages/ckeditor5-dev-tests/lib/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -8,23 +8,34 @@
 'use strict';
 
 const path = require( 'path' );
-const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
 const fs = require( 'fs-extra' );
+const _ = require( 'lodash' );
 const gutil = require( 'gulp-util' );
 const commonmark = require( 'commonmark' );
 const combine = require( 'dom-combiner' );
-const getRelativeFilePath = require( '../getrelativefilepath' );
-const _ = require( 'lodash' );
 const chokidar = require( 'chokidar' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const getRelativeFilePath = require( '../getrelativefilepath' );
+const globSync = require( '../glob' );
 
 const reader = new commonmark.Parser();
 const writer = new commonmark.HtmlRenderer();
 
-module.exports = function compileHtmlFiles( buildDir, manualTestPattern ) {
-	const globSync = require( '../glob' );
+/**
+ * @param {String} buildDir A path where compiled files will be saved.
+ * @param {Array.<String>} manualTestScriptsPatterns An array of patterns that resolve manual test scripts.
+ * @returns {Promise}
+ */
+module.exports = function compileHtmlFiles( buildDir, manualTestScriptsPatterns ) {
 	const viewTemplate = fs.readFileSync( path.join( __dirname, 'template.html' ), 'utf-8' );
-	const sourceMDFiles = globSync( path.join( manualTestPattern, '*.md' ) );
+
+	const sourceMDFiles = manualTestScriptsPatterns.reduce( ( arr, manualTestPattern ) => {
+		arr.push( ...globSync( manualTestPattern ).map( ( jsFile ) => setExtension( jsFile, 'md' ) ) );
```

Comment:
I'm not the fun of the Array reduce combined with Array push inside, because it's neither functional nor procedural programming. :D

Maybe it could be written with ES6 spread or concat:

```js
return [
   ...arr
   ...globSync( manualTestPattern ).map( jsFile => setExtension( jsFile, 'md' ) ),
]
```